### ISSUE TITLE
Hermitage skull-count displays incorrect/nonexistant

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#792]
0.7.8, win7x64, HD mode@1704x960windowed

I went to spend the bronze skull I knew I'd just earned, but only my stone skulls are showing at the hermit.

I think what may have happened was something to do with the post-battle screen.  When the popup came about earning a stone skull, I clicked Equip, and then Next.  It never gave me the second popup, so maybe it also didn't give me the bronze skull because of that.

I had spent all my stone ones, then beat another level, which I think would have earned me 4 stone skulls, then beat the catacombs.  So I believe it did give me the stone skull from there, just not the bronze.  I didn't even read it, but if the stone one comes first, that would make sense.

**Comment by IBwWG** [comment]
I tried to reproduce this by beating the catacombs by pressing F2, but then it didn't give any skull popup.  I guess you can't repeatedly earn them from a certain level.  But my last-posted save (in the journal issue maybe?) should work as a starting point.

**Comment by larsiusprime** [comment]
Please post your current save file?

I want to inspect whether the bronze skull is actually in there or not. It might just be that you don't have any upgradeable items that use bronze skulls.

**Comment by IBwWG** [comment]
[SAVE_SLOT_2.zip](https://github.com/larsiusprime/tdrpg-bugs/files/292251/SAVE_SLOT_2.zip)

**Comment by IBwWG** [comment]
I'm pretty sure I had a couple items that took bronze skulls.  I have all the upgradeable items, I think, and IIRC the last two swords took 1 and 2 respectively.

**Comment by larsiusprime** [comment]
Lesse here:

```
		<item type="skull" level="5" _class="nonequip" owner=""/>
		<item type="skull" level="5" _class="nonequip" owner=""/>
		<item type="skull" level="5" _class="nonequip" owner=""/>
		<item type="skull" level="1" _class="nonequip" owner=""/>
		<item type="skull" level="1" _class="nonequip" owner=""/>
		<item type="skull" level="1" _class="nonequip" owner=""/>
		<item type="skull" level="1" _class="nonequip" owner=""/>
		<item type="skull" level="2" _class="nonequip" owner=""/>
		<item type="skull" level="1" _class="nonequip" owner=""/>
```

That's 4 stone skulls (level 1), 1 bronze skull (level 2), and 3 gold skulls (level 5).

Does that sound like what you more or less should have?

If so, this is a display issue.

**Comment by IBwWG** [comment]
Yes, that seems accurate.

**Comment by IBwWG** [comment]
0.8.1, win7x64, HD mode@1704x960windowed

I spent all my stone skulls and 3 bronze ones.  Now I'm getting this strange behaviour:

1. on the bows tab, it says I have 0x skulls at the bottom
1. on the staffs tab, it says I have 3x
1. on the swords tab (no swords left to improve) it says whatever the last selected tab had said

Both the bows and staffs tab have items requiring bronze skulls available.

Why not just always show the count for all three skull types?

**Comment by IBwWG** [comment]
Actually the fact of 2. above is interesting, because I actually don't have any, I had only earned three, which I spent on sword upgrades.  Indeed, trying to spend these mystery skulls results in something strange:

![dq-cant-afford](https://cloud.githubusercontent.com/assets/16719964/15735409/eee3bd62-2898-11e6-87e3-81b8aa4f2b6e.png)

**Comment by ManDude** [comment]
What are gold skulls? Doesn't DQ only use stone, bronze and obsidian? I think the HUD is searching for gold skulls and not bronze skulls.

**Comment by IBwWG** [comment]
@ManDude I got the gold skulls from the first run through (non-NG+) from the place out in the desert with the runes.  There's info on the wiki about how to get them.

**Comment by ManDude** [comment]
Oh yes, the secret ones. I forgot about those.

**Comment by ManDude** [comment]
Alright, did some more testing, most likely a display bug.

- The game only shows 1 skull type, even if you have at least 2 different types of skulls in your inventory.
- The game will not update the skull icon when you switch tabs, so if you open the Weapons and all swords available require Obsidian skulls, you get the Obsidian skull icon stuck. You can then go to a different tab (say, bows) that has bows that require bronze skulls and press Equip and leave the party screen and you get the bronze skull icon stuck.

**Comment by larsiusprime** [comment]
Fixed in 0.8.9
